src/tests/makefile.all

```diff
@@ -381,24 +385,33 @@ NO_W32_CFLAGS = -D_Windows
     $(error Unknown WATCOM model. Only 'SMALL', 'LARGE', 'FLAT' and 'WIN32'.)
   @endif
 
-  COMMON_CFLAGS += -zq -wx -zm -d1 -I$(WATCOM)\h -I$(WATCOM)\h\nt
+  COMMON_CFLAGS += -zq -wx -zm -fo=.obj -d1 -I"$(WATCOM)/h" -I"$(WATCOM)/h/nt"
 
   #
-  # Watcom does not like '/' in it's arguments.
-  # 'wlink' generates this:
-  #   Error! E2030: file clib3s.lib(dstrt386): multiple starting addresses found, already defined in clib3s.lib(cstrtwnt)
+  # Watcom tools use '/' character as option prefix that items using '/'
+  # must be quoted.
```

Comment:
So Watcom tools is not getopt style? Is that the single reason for these `"-I../path"` options?

Reply:
No you can use by example -3r-s-d2  or -3r -s -d2 etc.

Reply:
Anyway on DOS-ish platform option prefix can be '-' (dash) or '/' slash on linux only '-' dash.